src/meta/src/manager/catalog/mod.rs

```diff
@@ -207,35 +328,28 @@ where
         let mut databases = BTreeMapTransaction::new(&mut database_core.databases);
         let mut schemas = BTreeMapTransaction::new(&mut database_core.schemas);
         databases.insert(database.id, database.clone());
-        let mut schemas_added = vec![];
-        for schema_name in iter::once(DEFAULT_SCHEMA_NAME).chain(SYSTEM_SCHEMAS) {
-            let schema = Schema {
-                id: self
-                    .env
-                    .id_gen_manager()
-                    .generate::<{ IdCategory::Schema }>()
-                    .await? as u32,
-                database_id: database.id,
-                name: schema_name.to_string(),
-                owner: database.owner,
-            };
-            schemas.insert(schema.id, schema.clone());
-            schemas_added.push(schema);
-        }
 
+        let schema = Schema {
+            id: self
+                .env
+                .id_gen_manager()
+                .generate::<{ IdCategory::Schema }>()
+                .await? as u32,
+            database_id: database.id,
+            name: DEFAULT_SCHEMA_NAME.to_string(),
```

Comment:
Seems in postgresql, public schema(DEFAULT_SCHEMA_NAME) in different database also will be the same. Do we need to fix here too?

Reply:
Yes, but the `public` schema is droppable and the id will change if re-create it. I'd prefer to create a new one for each database here.